osu.Game/Overlays/ModSelection/ModSelection.cs

```diff
@@ -0,0 +1,520 @@
+﻿//Copyright (c) 2007-2016 ppy Pty Ltd <contact@ppy.sh>.
+//Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK;
+using OpenTK.Graphics;
+using OpenTK.Input;
+using osu.Framework.Input;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Transformations;
+using osu.Game.Graphics.Backgrounds;
+using osu.Framework.MathUtils;
+using System.Collections.Generic;
+using osu.Framework.Graphics.Primitives;
+using osu.Game.Modes;
+using osu.Game.Graphics;
+
+namespace osu.Game.Overlays.ModSelection
+{
+    public class ModSelection : OverlayContainer
+    {
+        private FlowContainer easymods, hardmods, assistmods;
+        private SpriteText ranked, multiplier;
+        private PlayMode lastPlayMode;
+
+        private const float padded_width = 0.7f;
+        private const float mod_section_height = 0.31f;
+
+        private const int section_margin = -8;
+        private const int section_padding = 20;
+        private const int textsize_section_info = 16;
+        private const int textsize_multiplier_info = 40;
+
+        public ModSelection()
+        {
+            AddTemporaryMods();
+
+            RelativeSizeAxes = Axes.Both;
+            Anchor = Anchor.BottomLeft;
+            Origin = Anchor.BottomLeft;
+            Size = new Vector2(1.0f, 0.75f);

```

Comment:
you sure?

![image](https://cloud.githubusercontent.com/assets/191335/20998318/3d955532-bd50-11e6-8a18-b22b7a71805e.png)